pkg/utils/utils.go

```diff
@@ -162,3 +164,34 @@ func RemoveHyperlink(hyperlink string) string {
 	}
 	return command
 }
+
+// IsNamespaceAllowed checks if a given Namespace is allowed based on the config.
+func IsNamespaceAllowed(nsCfg config.Namespaces, givenNs string) bool {
+	isAll := len(nsCfg.Include) == 1 && nsCfg.Include[0] == config.AllNamespaceIndicator
```

Comment:
that's an existing functionality, personally I would no features, let's do that in the dedicated PR

Reply:
😶 Mixed feelings there.
With the regex I agree, it would need to be documented, let's keep it for later. But here? This comment was not about a new feature, but more like a bug. If `all` is specified, all namespaces should be considered. The only thing I was asking for is the line here:


And such unit test case:

```go
		"should watch all except ignored ones if there are more included namespaces": {
			nsConfig:  config.Namespaces{Include: []string{"all", "single"}, Ignore: []string{"demo"}},
			givenNs:   "abc",
			isAllowed: true,
		},
```


No docs changes are needed, it's just a hidden, user-proof behavior.

Reply:
😶 Mixed feelings there.
With the regex I agree, it would need to be documented, let's keep it for later. But here? This comment was not about a new feature, but more like a bug. If `all` is specified, all namespaces should be considered. The only thing I was asking for is the line here:

```go
isAll := slices.Contains(n.Include, AllNamespaceIndicator)
```

And such unit test case:

```go
		"should watch all except ignored ones if there are more included namespaces": {
			nsConfig:  config.Namespaces{Include: []string{"all", "single"}, Ignore: []string{"demo"}},
			givenNs:   "abc",
			isAllowed: true,
		},
```

No docs changes are needed, it's just a hidden behavior that is more user-proof. Can you apply that please?